crates/turborepo-lib/src/shim.rs

```diff
@@ -388,8 +476,8 @@ pub fn run() -> Result<Payload> {
         Err(err) => {
             // If we cannot infer, we still run global turbo. This allows for global
             // commands like login/logout/link/unlink to still work
-            eprintln!("Repository inference failed: {}", err);
-            eprintln!("Running command as global turbo");
+            debug!("Repository inference failed: {}", err);
```

Comment:
I really don't know how I feel about all of these messages being behind `debug` regardless of the command. The alternative of having special cases on commands that shouldn't output this information. e.g. `bin` or `--version` also feels brittle.

I think this will be less of an issue once we have `turboVersion` in `turbo.json` be the source of truth as the global/local turbo divide will more obvious.

Reply:
Yea I know what you mean. Honestly I think we wouldn't want to see these by default for _almost everything_. The one real exception being `run`. So maybe we just go the other direction and allowlist it